README.md

```diff
@@ -103,6 +103,15 @@ find(
 ): Promise<PagedResult<Schema>>;
 ```
 
+## 8.0.0 Upgrade Guide
+
+### Double Encoding
```

Comment:
I know I said I didn't want a book, but I do belief we can elaborate a bit more like you did in the inline comment in the deprecation. I would even suggest putting most of that message here and just a small description and reference to here.

Would it make sense to start of by stating something like "From now on values supplied to RQL operations are now automatically encoded to be safe to be used in RQL. Therefore any (URL) encoding of these values currently implemented should be removed."?

And suggesting if this is not feasible you could disable globally and optionally enable on each call. But like you said, that should be something temporary.

Maybe this would make sense to be discussed like documentation as well 😅